docs/nodes/GENERATORS/SIMULATIONS/RAND/RAND.md

```diff
@@ -1,3 +1,10 @@
+---
+title: RAND
+description: The RAND node generates a random number or a list of random numbers depending on the distribution selected. Master Python"-"powered random simulations with FloJoy's RAND node for seamless and efficient random simulations.
```

Comment:
'FloJoy' -> 'Flojoy'
Also are the quotes around the dash necessary? I assume they're for making it a literal dash instead of interpreting it as markdown?